<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1447</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QSlider" name="seekBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>971</width>
      <height>20</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forward">
    <property name="geometry">
     <rect>
      <x>572</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="currentSong">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>881</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 12pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Song</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>50</y>
      <width>401</width>
      <height>421</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="movement">
     <enum>QListView::Static</enum>
    </property>
    <property name="viewMode">
     <enum>QListView::ListMode</enum>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="repeat">
    <property name="geometry">
     <rect>
      <x>658</x>
      <y>490</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffle">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="volumeBar">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>496</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider{
	color:white;

}</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mute">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imgMusic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>871</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangePassword">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userOnl">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>480</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 12pt &quot;Sitka Display Semibold&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string>User:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListUser">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>350</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProfile">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>300</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addMusic">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteMusic">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>150</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="editMusic">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bgContainer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>2391</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Image/Hinh-Nen-Trang-10.webp</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>530</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="bgContent">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>0</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Image/Hinh-Nen-Trang-10.webp</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="selectCategory">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 10pt &quot;Sitka Display Semibold&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReload">
    <property name="geometry">
     <rect>
      <x>1372</x>
      <y>20</y>
      <width>24</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius:10px;
	background:white;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="userInfo">
    <property name="geometry">
     <rect>
      <x>1149</x>
      <y>480</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 12pt &quot;Sitka Display Semibold&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>520</y>
      <width>130</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 12pt &quot;Sitka Display Semibold&quot;;
color:black;
</string>
    </property>
    <property name="text">
     <string>Role:</string>
    </property>
   </widget>
   <widget class="QLabel" name="roleInfo">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>520</y>
      <width>260</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 italic 12pt &quot;Sitka Display Semibold&quot;;
color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>bgContent</zorder>
   <zorder>imgMusic</zorder>
   <zorder>bgContainer</zorder>
   <zorder>btnListUser</zorder>
   <zorder>btnProfile</zorder>
   <zorder>addMusic</zorder>
   <zorder>deleteMusic</zorder>
   <zorder>editMusic</zorder>
   <zorder>seekBar</zorder>
   <zorder>back</zorder>
   <zorder>play</zorder>
   <zorder>forward</zorder>
   <zorder>currentSong</zorder>
   <zorder>listWidget</zorder>
   <zorder>repeat</zorder>
   <zorder>shuffle</zorder>
   <zorder>volumeBar</zorder>
   <zorder>mute</zorder>
   <zorder>btnLogout</zorder>
   <zorder>btnChangePassword</zorder>
   <zorder>userOnl</zorder>
   <zorder>label</zorder>
   <zorder>selectCategory</zorder>
   <zorder>searchBar</zorder>
   <zorder>btnReload</zorder>
   <zorder>userInfo</zorder>
   <zorder>label_3</zorder>
   <zorder>roleInfo</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
